src/managers/auth-manager.js

```diff
@@ -70,6 +85,32 @@ var _findUserInGroup = function (members, id) {
 };
 
 AuthManager.prototype = $.extend(AuthManager.prototype, {
+
+    /**
+    * Logs user in.
+    *
+    * For [end users](../../../glossary/#users), the `group` is required in order to log in. You can pass it as part of the `options` parameter if it was not included when the Authorization Manager was instantiatied. 
+    *
+    * If some end users are in multiple groups, on the login page for your project you can call `getUserGroups()` (see details below), present the end users with a list of groups, and require that they select one.
+    *
+    * **Example**
+    *
+    *       authMgr.login({
+    *           account: 'acme-simulations', 
+    *           userName: 'enduser1', 
+    *           password: 'passw0rd' 
+    *       });
+    *
+    *       authMgr.login()
+    *           .then(function(authAdapter) {
+    *               authMgr.getUserGroups({userId: authAdapter.user.user_id});
```

Comment:
Based on #60 this shouldn't work.

```
authMgr.login().then(function () { // will only work if you're in 1 group}).fail(function () { //Can fail for multiple reasons, invalid passwords, multiple groups etc.});
``

getUserGroups isn't that relevant here. You _could_ get it that way, but otherwise the 'fail' response will already have the groups in it.